### ISSUE TITLE
Message attachement to base64 instead url

### ISSUE THREAD

**Issue by FrancisLaTruelle** [Androz2091/discord-backup#163]
Hello, is it possible to save the images of messages sent in base64 (like for emojis but for attachments to messages) ? And why not add an option to deactivate or activate the base64 for attachements. It makes very heavy backups but very useful for me please :)

**Comment by SimonLeclere** [comment]
The option to choose how to save the images already exists. 

> saveImages: How to save images like guild icon and emojis. Set to "url" by default, restoration may not work if the old server is deleted. So, url is recommended if you want to clone a server (or if you need very light backups), and base64 if you want to backup a server. Save images as base64 creates heavier backups.

However, I don't think that attachment backup is implemented.

**Comment by Androz2091** [comment]
Yes, indeed we could implement a way store attachements using base64 (or at least strings, using a different encoding) but the backups could become really really heavy. But yes, we could implement this and turn it off by default.
